### PR TITLE
Dart/Flutter SDK Page Revamp

### PR DESCRIPTION
**Pull request by emmaodia**

### CODE REVIEW

File: docs/developers/fuse-box/flutter-sdk/4-send-batch-txn.mdx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+---
+
+### Non-Blocking Transactions
+
+A batch refers to a group of calls inside a single user operationhe. This can be an approve and send transaction in one call. The executeBatch() method is used.
+
+The executeBatch() method takes in an array of _`Call()`_. With each method taking in 3 keys: to, value and data.
+
+```javascript
+executeBatch([
```
Comment (LiorAgnin) [inline docs/developers/fuse-box/flutter-sdk/4-send-batch-txn.mdx:13]: @emmaodia That's incorrect. In Dart it should be:

```dart
.executeBatch([
    Call(to: to, value: value, data: data),
    Call(to: to, value: value, data: data),
    Call(to: to, value: value, data: data),
  ]);
```

File: docs/developers/fuse-box/flutter-sdk/4-send-batch-txn.mdx [hunk 2]
```diff
@@ -0,0 +1,79 @@
+---
+title: Send Batch Transactions
+sidebar_position: 4
+---
+
+### Non-Blocking Transactions
```
Comment (LiorAgnin) [inline docs/developers/fuse-box/flutter-sdk/4-send-batch-txn.mdx:6]: Non-Blocking Transactions is a feature that can be used in both single & batch transactions.
It needs a separated guide.
You can read about the feature here: https://docs.stackup.sh/docs/useroperation-nonce#why-use-2-dimensional-nonces

File: docs/developers/fuse-box/flutter-sdk/2-create-scw.mdx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+### Initialization
+
+The `init` method is a fundamental feature of the Fuse Wallet Flutter SDK that enables you to setup a wallet & authenticate the user's provided credentials. When you send a request to the server using this method, it verifies the credentials and if the credentials are valid, developers can use the **`getSender()`** to return an address.
+
+```dart
+// Generate a random Ethereum private key
+final String privateKey = await Mnemonic.generatePrivateKey();
+final EthPrivateKey credentials = EthPrivateKey.fromHex(privateKey);
+
+final fuseSDK = await FuseSDK.init(credentials);
```
Comment (LiorAgnin) [inline docs/developers/fuse-box/flutter-sdk/2-create-scw.mdx:15]: @emmaodia The first argument needs to be the apiKey of the operator. Please fix it.

File: docs/developers/fuse-box/flutter-sdk/4-send-batch-txn.mdx (hunk 1 above)
Comment (emmaodia) [reply docs/developers/fuse-box/flutter-sdk/4-send-batch-txn.mdx:13]: Fixed.

File: docs/developers/fuse-box/flutter-sdk/2-create-scw.mdx (hunk 3 above)
Comment (emmaodia) [reply docs/developers/fuse-box/flutter-sdk/2-create-scw.mdx:15]: Fixed.

File: docs/developers/fuse-box/flutter-sdk/4-send-batch-txn.mdx (hunk 2 above)
Comment (emmaodia) [reply docs/developers/fuse-box/flutter-sdk/4-send-batch-txn.mdx:6]: Moved Non-Blocking to a separate file.

**Comment by emmaodia** [comment]
Feedback has been implemented, and comments have been resolved. @LiorAgnin

File: docs/developers/fuse-box/flutter-sdk/4-send-batch-txn.mdx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+```
+
+| Parameter | Type       | Status     | Description                                      |
+| :-------: | ---------- | ---------- | ------------------------------------------------ |
+|  **to**   | _address_  | _required_ | The address where the user want to send Tokens.  |
+| **value** | _number_   | _required_ | The amount of Tokens the user is sending         |
+| **data**  | _Function_ |            | Encoded Function Data for Smart Contract Methods |
+
+Example **`transfer_and_approve.dart`**
+
```
Comment (LiorAgnin) [inline docs/developers/fuse-box/flutter-sdk/4-send-batch-txn.mdx:27]: The example isn't incorrect. It doesn't make sense to transfer and approve. It needs to be approve and transfer.
Secondly, the code example needs to be updated accordingly. Here's an example:

```
  final res = await fuseSDK.executeBatch(
    [
      // Approve ERC20 Token call
      Call(
        to: tokenAddress,
        value: BigInt.zero,
        data: ContractsUtils.encodeERC20ApproveCall(
          tokenAddress,
          recipientAddress,
          amountInWei,
        ),
      ),
      // Transfer ERC20 Token call
      Call(
        to: tokenAddress,
        value: BigInt.zero,
        data: ContractsUtils.encodeERC20TransferCall(
          tokenAddress,
          recipientAddress,
          amountInWei,
        ),
      ),
    ],
  );
  print('UserOpHash: ${res.userOpHash}');

  print('Waiting for transaction...');
  final ev = await res.wait();
  print('Transaction hash: ${ev?.transactionHash}');
```

Comment (emmaodia) [reply docs/developers/fuse-box/flutter-sdk/4-send-batch-txn.mdx:27]: Great! Thanks for catching that. Fixed.

File: docs/developers/fuse-box/flutter-sdk/19-non-blocking-txn.mdx [hunk 5]
```diff
@@ -0,0 +1,74 @@
+---
+title: Non-Blocking Transactions
+sidebar_position: 19
```
Comment (LiorAgnin) [inline docs/developers/fuse-box/flutter-sdk/19-non-blocking-txn.mdx:3]: @emmaodia IMO it should be after 5.

Comment (emmaodia) [reply docs/developers/fuse-box/flutter-sdk/19-non-blocking-txn.mdx]: Noted. Fixed.

**Pull request merged**